### DISCUSSION TITLE
Introducing a user/oversight board on top of the project

### DISCUSSION THREAD

**Discussion by insertish** [stoatchat/discussions#767] [Focused Discussions]
As mentioned in the [August Moth](https://revolt.chat/posts/month-in-revolt-august-2024), we are working on changing the organisational/project structure, and one of the things I would like to implement is a sort of 'user board' that sits above leadership, can peer into how the project is working, and can provide advice for the team, contest decisions made by the team, and so forth. It would consist of people who can represent the user-base to communicate with the developers.

**Comment by insertish** [comment]
Some things in no particular order brainstormed with people: (not saying any of these are a good idea)
- The user board should create its own mechanisms for taking opinions from the community, which the Revolt team can support, and can discuss and work together on suggestions/solutions (as its own thing).
- Can pull in owners of big servers, or better: allow them to nominate a representative; because we want someone who will actively participate.
- > if you don't set some sort of constitution with minimum conditions for x and y
  > that is going to devolve into a closed circle of one kind of another

Some things that need to be considered:
- How would the user board be organised and talk to each other?
- How are people elected on to it?
- How do we ensure users can be heard?

**Reply by gohellp** [reply]
> How would the user board be organised and talk to each other?

A private server that only certain people can join. But this you need a certain infrastructure which can allow/deny user from join' by his's ID.

> How are people elected on to it?
> How do we ensure users can be heard?

I think it should be chosen by community vote or something. These users can collect some topics from the server and redirect them to the Team.

MB you can implement some kind of ticket system for this

**Reply by Shanesan** [reply]
The problem with the community voting on anything is that, usually, the community has no idea what they _actually want or need_ because they aren't programmers or UX people - they're consumers, and beyond "I want chat, video, screen-sharing, and a good interface" it becomes a popularity contest which is something you typically don't want to have when it comes to real work needing to be done.

Saying that, the Board can look at these things and filter them or promote them, like, a bunch of ideas come in, some of them are less 'loud' than they should be, the board can mark them as important and maybe drive some community involvement.

So you got a "voting population" and a "social board" which can work in tandem or against each other to balance what the Board believes is needed but may not be socially understood/popular ([Federation](https://github.com/orgs/revoltchat/discussions/469)? A lean towards [self-hosting support](https://github.com/orgs/revoltchat/discussions/846)?) vs what the community wants on the surface ([Video chat](https://github.com/orgs/revoltchat/discussions/196), [Screensharing](https://github.com/orgs/revoltchat/discussions/178)).

**Comment by DeclanChidlow** [comment]
I love August Moth

**Comment by Nasher73** [comment]
How far did you go with that oversight board idea ?
